specification.md

```diff
@@ -202,39 +202,38 @@ Event data are stored in CSV files. Simple event data should be
 stored in a single column CSV, with each element in the array indicating the time of the
 event **relative to the start of the dataset**. The first line of the file must contain `start,`,
 indicating that the column contains the times of the event data. Event datasets can be
-distinguished from sampled datasets because the file is a plaintext CSV and `units` attribute must be
```

Comment:
Additionally, that the 'units' distinction serves this compatibility purpose is probably useful information to put in the spec.